tech/languages/rust/rust-installation.md

```diff
@@ -9,10 +9,25 @@ description: Systems programming language that runs blazingly fast, prevents seg
 
 # Rust installation
 
-In order to get started with Rust programming, type:
+## Prepackaged Rust
+
+Fedora's repositories offer prepackaged versions of Rust and its package manager `cargo`, which can be installed as follows:
+
 ```
 $ sudo dnf install rust cargo
 ```
-This will install the compiler (`rustc`), standard library, gdb support, documentation generator (`rustdoc`) and the package manager (`cargo`).
 
-Continue to the next section where project creation is discussed.
+This command will install the `rustc` compiler, the standard library, gdb support, the `rustdoc` documentation generator and the package manager.
+
+## Via Rust's own installer
+
+Rust's [official installation instructions](https://www.rust-lang.org/learn/get-started) recommend its own installation mechanism, `rustup`. Rust's official installer is available via `dnf install rustup`. You can run `rustup-init` after installing the package to start the Rust installation.
+
+## Which one do I need?
+
+It depends on your preferences. Some people may prefer prepackaged Rust because it automatically updates via `dnf`. Others may prefer a standardized installation. If you go for prepackaged Rust, keep this in mind:
+
+- `cargo clippy` is not included in Fedora's `cargo` package, but is available via the `clippy` package.
+- To enable autocompletions and other code intelligence features in your IDE, you may need to install the `rust-src` and `rustfmt` packages.
+
+In the next section project creation is discussed.
```

Comment:
Hmm, this reads a bit weird, especially when I ask myself where is this next section exactly. I think we can just leave the page without it.

```suggestion
```